File: exercises/day14-Santa-package-factory-exercises/csharp/PackagingService/Gift.cs
namespace PackagingService
{
    public class Gift
    {
        public string Name { get; }
        public GiftSize Size { get; }
        public bool IsFragile { get; }
        public int RecommendedMinAge { get; }
        
        public Gift(string name, GiftSize size, bool isFragile, int recommendedMinAge)
        {
            Name = name;
            Size = size;
            IsFragile = isFragile;
            RecommendedMinAge = recommendedMinAge;
        }
    }
}

File: exercises/day14-Santa-package-factory-exercises/java/src/main/java/packagingservice/Gift.java
package packagingservice;

public class Gift {

    private final String name;
    private final GiftSize size;
    private final boolean fragile;
    private final int recommendedMinAge;

    public Gift(String name, GiftSize size, boolean isFragile, int recommendedMinAge) {
        this.name = name;
        this.size = size;
        this.fragile = isFragile;
        this.recommendedMinAge = recommendedMinAge;
    }

    public String getName() {
        return name;
    }

    public GiftSize getSize() {
        return size;
    }

    public boolean isFragile() {
        return fragile;
    }

    public int getRecommendedMinAge() {
        return recommendedMinAge;
    }
}
